src/components/TextField/README.md

```diff
@@ -830,32 +830,30 @@ See Apple’s Human Interface Guidelines and API documentation about accessibili
 
 <!-- content-for: web -->
 
-### Structure
-
 Screen readers convey information about text fields automatically through native HTML.
 
 - Use the `disabled` prop to add the HTML `disabled` attribute to the text field.
 - Use the `readOnly` prop to add the HTML `readonly` attribute to the text field.
-- If you use the `type` prop, then some assistive technologies adapt the software keyboard to the current task. This helps merchants with mobility, vision, and cognitive issues to enter information more easily.
+- If you use the `type` prop, then some browsers and assistive technologies adapt the software keyboard to the current task. This helps users with mobility, vision, and cognitive issues to enter information more easily.
```

Comment:
With the introduction of "browsers", does "software keyboard" still cover all use cases?

Reply:
I think so. I was trying to get at software keyboards on mobile, as well as those introduces by things like switch devices. This can probably be clearer, though. 🤔